src/SDKs/SqlManagement/Sql.Tests/ServerKeyScenarioTests.cs

```diff
@@ -53,12 +55,13 @@ public void TestCreateUpdateDropServerKey()
                 var keyList = sqlClient.ServerKeys.ListByServer(resourceGroup.Name, server.Name);
                 Assert.Equal(2, keyList.Count());
 
-                // Delete key
-                sqlClient.ServerKeys.Delete(resourceGroup.Name, server.Name, serverKeyName);
+                //TODO: Temporarily disabling this since delete operation is affected by a production bug.
```

Comment:
What is the production bug? We should avoid publishing an SDK that has known issues.

Reply:
Hi Jared, 

The bug is in management service where the post condition checks are failing even on operation success. This issue seems to affect multiple APIs in prod. Management service team is currently investigating the issue. 
It is important to note that the failure occurs in an existing functionality (sqlClient.ServerKeys.Delete() method which was released last year) and is not caused by this change.

Reply:
Not related to managed instance keys.